From db78edb2e3f0a60317b167b3a4002d0f5aa5a2a7 Mon Sep 17 00:00:00 2001
From: Artem Titov <titovartem@chromium.org>
Date: Mon, 7 May 2018 13:13:08 +0200
Subject: [PATCH] Fix protobuf's library .gitinore file. Fix .gitignore file
 for protobuf library not to exclude file, that is checkid in into repo.

---
 third_party/protobuf/.gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/protobuf/.gitignore b/third_party/protobuf/.gitignore
index 51d092c72475..a130b8bcc028 100644
--- a/third_party/protobuf/.gitignore
+++ b/third_party/protobuf/.gitignore
@@ -53,6 +53,7 @@ src/google/protobuf/util/**/*.pb.h
 *.pyc
 *.egg-info
 *_pb2.py
+!python/google/protobuf/descriptor_pb2.py
 python/*.egg
 python/.eggs/
 python/.tox
-- 
2.17.0.441.gb46fe60e1d-goog

